ndmg/graph/gen_graph.py

```diff
@@ -290,14 +296,12 @@ def save_graph_png(self, graphname):
         import matplotlib
         matplotlib.use('agg')
         from matplotlib import pyplot as plt
-        from nilearn.plotting import plot_matrix
+        from graspy.utils import ptr
+        from graspy.plot import heatmap
 
-        from sklearn.preprocessing import normalize
         conn_matrix = np.array(nx.to_numpy_matrix(self.g))
```

Comment:
I'm wondering if this is where the sorting discrepancy between our plots happens. I'm using graspy's [`import_edgelist`](https://github.com/neurodata/graspy/blob/master/graspy/utils/utils.py#L152), which contains these lines:

```
    vertices = np.sort(reduce(np.union1d, [G.nodes for G in graphs]))
    out = [nx.to_numpy_array(G, nodelist=vertices, dtype=np.float) for G in graphs]
```
this sorts by vertex number

Reply:
Interesting.

Here's an alternative way:
```
import networkx as nx
import numpy as np

edgelist_path = '/tmp/output_desikan_det_local_csd_native/dwi/roi-connectomes/desikan_space-MNI152NLin6_res-2x2x2/sub-0025427_ses-1_dwi_desikan_space-MNI152NLin6_res-2x2x2_measure-spatial-ds_adj.ssv'

g_arr = np.loadtxt(open(edgelist_path, "rb"), delimiter=" ")

G = nx.Graph()

for i in g_arr:
    [edge_i, edge_j, weight] = i
    G.add_edges_from([(edge_i, edge_j)], weight=weight)

conn_matrix = nx.to_numpy_matrix(G)
```

Reply:
We give an arbitrary sorting to the nodes because the way you make the graph here doesn't guarantee that the first element of the `conn_matrix` correspond to the same nodes across multiple edgelists. `nx.to_numpy_matrix()` just builds the array based on the order of the nodes that `g_arr` is in, which is arbitrary node ordering.

Reply:
Good to know!